src/picongpu/submit/pizdaint-cscs/SLURM_Tutorial.md

```diff
@@ -0,0 +1,27 @@
+SLURM examples
```

Comment:
you can just add a softlink to `taurus-tud/SLURM_Tutorial.md` as we did in `lawrencium-lbnl/`.